js/src/tooltip.js

```diff
@@ -41,6 +41,7 @@ const DefaultType = {
   container: '(string|element|boolean)',
   fallbackPlacement: '(string|array)',
   boundary: '(string|element)',
+  additionalClasses: '(string|function)',
```

Comment:
I would prefer a parameter called : `customClass` which would allow an array of string or a function it would be better if someone wants to add a lot of classes

Reply:
@Johann-S  As this PR stands now, the additionalClasses property is a logic-less wrapper around [`jQuery.addClass()`](https://api.jquery.com/addClass/#addClass-classNames). As of jquery 3.3, addClass() does accept an array of classes. So my initial thinking was that in Bootstrap 4.X, we guarantee that a string or function are valid inputs (although passing in an array will work in environments with a modern version of jQuery), and then in Bootstrap 5 we could rely on a more modern version of jQuery. But I just checked `package.json` on the `main` branch, and jQuery is no longer listed as a peer dependency. Will Bootstrap 5 not depend on jQuery? If so I will definitely change my strategy.

Reply:
in v4 we can stay like you did, just the renaming (additionalClasses -> customClass) but since v5 is intended to work without jQuery I would prefer an array and a function